app/event/raise-event.js

```diff
@@ -0,0 +1,28 @@
+const { PublishEvent } = require('ffc-ahwr-event-publisher')
+const { eventQueue } = require('../config').mqConfig
+
+const raiseEvent = async (event, status = 'success') => {
+  const eventPublisher = new PublishEvent(eventQueue)
+
+  const eventMessage = {
```

Comment:
if it's a monitoring event it'd be best to stick to this name and be very consistent about it.

raise-event.js file please rename to raise-monitoring-event.js

the folder please rename from event to monitoring-event